data/json/itemgroups/food_service.json

```
@@ -976,5 +976,49 @@
       { "item": "stout", "prob": 10, "container-item": "keg" },
       { "item": "imperial_stout", "prob": 5, "container-item": "keg" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "butcher_raw_meat",
+    "items": [
+      [ "meat", 8 ],
+      [ "lunchmeat", 6 ],
+      [ "offal", 10 ],
+      [ "hotdogs_frozen", 6 ],
+      { "item": "lung", "prob": 1, "charges": 2 },
+      { "item": "kidney", "prob": 1, "charges": 2 },
+      { "item": "liver", "prob": 1, "charges": 1 },
+      { "item": "brain", "prob": 1, "charges": 1 },
+      { "item": "sweetbread", "prob": 2, "charges": 1 }
```

```suggestion
      { "item": "sweetbread", "prob": 2, "charges": 2 }
```
Sweetbread gets fewer charges for balance reasons. Calories on it are enough that you can make good meals out of the sweetbread instead of using it in something else.